SynthSeg/training_options.py

```diff
@@ -368,6 +368,24 @@ class TrainingOptions(Serializable, OptionsBase):
     See https://www.tensorflow.org/guide/distributed_training for more information. 
     """
 
+    scaling_type: str = "weak"
+    """
+    Possible values: "weak", "strong". 
+    Option that was used when profiling the distributed training. 
+    When set to "strong" will set batch_size to the batch_size/num_gpu (to see same amount of data as withg one gpu). 
+    When set to "weak" batch size will stay the same.
+    """
+
+    find_last_checkpoint: bool = True
```

Comment:
I would prefer to set this as `False` to not change the current default behavior.

Reply:
done